packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -930,6 +930,14 @@ export const Table = <
                   {headerGroup.headers.map((column, index) => {
                     const { onClick, ...restSortProps } =
                       column.getSortByToggleProps();
+
+                    const columnHasExpanders =
+                      hasAnySubRows &&
+                      index ===
+                        headerGroup.headers.findIndex(
+                          (c) => c.id !== SELECTION_CELL_ID, // first non-selectable column is the expander column
```

Comment:
nit: Since columns are not selectable. The column's name is selection column.
```suggestion
                          (c) => c.id !== SELECTION_CELL_ID, // first non-selection column is the expander column
```